server/src/main/java/org/cloudfoundry/identity/uaa/zone/MergedZoneBrandingInformation.java

```diff
@@ -33,6 +33,14 @@ public String getProductLogo() {
         return tryGet(IdentityZoneHolder.get(), BrandingInformationSource::getProductLogo).orElse(null);
     }
 
+    public String getInlinedBase64PngLogo() {
+        String logo = getProductLogo();
```

Comment:
Especially since this one appears to already be in base64, despite its name not reflecting that.